netmiko/base_connection.py

```diff
@@ -955,6 +1190,24 @@ def send_config_set(self, config_commands=None, exit_config_mode=True, delay_fac
         The commands will be executed one after the other.
 
         Automatically exits/enters configuration mode.
+
+        :param config_commands: Multiple commands to be sent to the device
+        :type config_commands: iterable of strings
```

Comment:
Nesting types using `of` is supported by AFAIK at least PyCharm. `list of str` would be valid for instance and is correctly parsed by the PyCharm static type checker.